aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -318,6 +320,7 @@ void f8f8bf16_rowwise_impl_sm100(
     std::optional<at::Tensor> bias,
     at::Tensor out,
     const int swizzle) {
+#if (defined(__CUDA_ARCH__)) && (__CUDA_ARCH__ == 1000 || __CUDA_ARCH__ == 1030)
```

Comment:
Here it is more complicated. We could get away with `__CUDA_ARCH_FAMILY_SPECIFIC__` only if we compile with `sm_100f`.

Reply:
I believe we can re-use cutlass macros here: https://github.com/NVIDIA/cutlass/blob/5b76420d6ae0ec0dbf82dc19317890551bffb1a6/include/cutlass/detail/helper_macros.hpp#L211-L236

Reply:
Sorry for the noise, I'm not sure these are still needed because it looks like CUTLASS will implicitly disable code depending on CUDA arch